icons/circle-off.svg

```diff
@@ -10,6 +10,6 @@
   stroke-linejoin="round"
 >
   <path d="m2 2 20 20" />
-  <path d="M8.4 2.7c1.2-.4 2.4-.7 3.7-.7 5.5 0 10 4.5 10 10 0 1.3-.2 2.5-.7 3.6" />
-  <path d="M19.1 19.1C17.3 20.9 14.8 22 12 22 6.5 22 2 17.5 2 12c0-2.7 1.2-5.2 3-7" />
+  <path d="M8.4 2.7a10 10 0 0 1 13 12.9" />
+  <path d="M19.1 19.1A9 9 0 1 1 5 5" />
```

Comment:
@jguddas: I'm afraid your optimization script still needs some tweaking, we've once again lost pixel perfection. 😞
```suggestion
  <path d="M19.08 19.08A10 10 0 1 1 4.92 4.92" />
  <path d="M8.35 2.69A10 10 0 0 1 21.3 15.65" />
```
![image](https://github.com/lucide-icons/lucide/assets/17746067/e4f8a682-210f-4cec-a61e-2d8741e05de1)